backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -419,4 +420,13 @@ class TestOperator extends WholeStageTransformerSuite {
         }
       }
   }
+
+  test("test OneRowRelation") {
+    val df = sql("SELECT 1")
+    checkAnswer(df, Row(1))
+    val plan = df.queryExecution.executedPlan
+    assert(plan.find(_.isInstanceOf[RDDScanExec]).isDefined)
+    assert(plan.find(_.isInstanceOf[ProjectExecTransformer]).isDefined)
+    assert(plan.find(_.isInstanceOf[GlutenRowToArrowColumnarExec]).isDefined)
```

Comment:
There's a refactor on class name on main branch
```suggestion
    assert(plan.find(_.isInstanceOf[RowToArrowColumnarExec]).isDefined)
```